### ISSUE TITLE
Design of 1-4 rows of pegs

### ISSUE THREAD

**Issue by amanda-phet** [phetsims/plinko-probability#99]
When there are only 3 or 4 rows of pegs, the size of the pegs and balls looks good to me. 

![4-rows](https://cloud.githubusercontent.com/assets/7852916/26801526/6466ef72-49fa-11e7-8235-ebcb392abe55.png)
![3-rows](https://cloud.githubusercontent.com/assets/7852916/26801519/601b4ac6-49fa-11e7-9fcd-7a24affea8db.png)

However, at 1 and 2 rows, the peg seems unusually large. 
![2-rows](https://cloud.githubusercontent.com/assets/7852916/26801537/6b234d10-49fa-11e7-80e4-b9da428b87f2.png)
![1-row 1](https://cloud.githubusercontent.com/assets/7852916/26801554/76e34542-49fa-11e7-8b19-f1176328a168.png)

I think the peg needs to max out at a certain size. Can you make the peg size for 1-2 rows match the peg size at 3 rows? So, the peg would not grow in size below 3 rows of pegs.

**Comment by veillette** [comment]
![plinko probability screenshot 8](https://user-images.githubusercontent.com/7633344/26850107-675a8a1a-4ad4-11e7-9839-bee9bbac24ff.png)
![plinko probability screenshot 7](https://user-images.githubusercontent.com/7633344/26850108-684bed42-4ad4-11e7-85f7-cae55e9929b8.png)
![plinko probability screenshot 6](https://user-images.githubusercontent.com/7633344/26850110-6938db20-4ad4-11e7-98bf-fc0afff2c8f3.png)

**Comment by amanda-phet** [comment]
The peg seems pretty small. Are you sure that is the size at 3 rows? It looks like the size at 4 rows, to my eye. 

Perhaps we should just focus on the 1-row case. This is the mockup from April. Can get close to this size for 1 row, and see how the others scale after that? I'm not sure what kind of scaling logic you are using, though.

![screen shot 2017-06-06 at 2 32 37 pm](https://user-images.githubusercontent.com/7852916/26850545-22a36b12-4ac5-11e7-8267-901d15136fa8.png)

**Comment by veillette** [comment]
@amanda-phet: Row 1 and 2 are essentially custom made so we can adjust the radius of the balls, the radius of the pegs and the the vertical offset of the pegs easily.
I'll use your screenshot from row 1 and adjust row 2 accordingly.

**Comment by amanda-phet** [comment]
That sounds great.

**Comment by veillette** [comment]
![plinko probability screenshot 13](https://user-images.githubusercontent.com/7633344/26928175-11584e2c-4c23-11e7-9679-a2e83cf7d13a.png)
![plinko probability screenshot 14](https://user-images.githubusercontent.com/7633344/26928236-67862b02-4c23-11e7-9a6e-d1e89b9d20dd.png)
![plinko probability screenshot 15](https://user-images.githubusercontent.com/7633344/26928270-836071de-4c23-11e7-860a-1c37f0ad9123.png)

**Comment by veillette** [comment]
@amanda-phet , based on your screenshort, I rebased the scale of the balls as well. 

The pegs are rescaled according to be inversely proportional to the number of bins rather than the number of rows
The pegs size is proportional to 1/(rows+1). This affects the scaling for all rows but is is hard noticeable for larger number or rows, but it makes a larger difference when the number of rows is small. 

Currently for rows 3-26, the balls are bigger than the pegs, but for rows 1 and 2 and three  the balls are scaled down.
For Row=1, the ball size is reduced by 75% of what is would be according to our simple scaling, 
For Row=2, the ball size is reduced by 80% of what is would be according to our simple scaling, 
For Row=3, the ball size is reduced by 86% of what is would be according to our simple scaling, 

For row four and above it follow the simple scaling rule.

I have included screenshots above.
